### PR TITLE
[4.0]Changed the helper text in Redirect plugins Advanced settings

### PR DESCRIPTION
**Pull request by mrsgv**
Pull Request for Issue #32793.

### Summary of Changes
Added a helper text "Only status codes from 300-399 correspond to a redirect.", So that the user inputs the correct status code.

### Testing Instructions

- Login as administrator
- Go to System -> Redirects -> New/ Edit.
- The help text can be seen below.

### Actual result BEFORE applying this Pull Request
111921232-64bb0f00-8a8b-11eb-8ba5-72dac816b0af

### Expected result AFTER applying this Pull Request

Updated status code

### CODE REVIEW

File: administrator/language/en-GB/com_redirect.ini [hunk 1]
```diff
@@ -33,7 +33,7 @@ COM_REDIRECT_FIELD_NEW_URL_DESC="Enter the URL to be redirected to."
 COM_REDIRECT_FIELD_NEW_URL_LABEL="New URL"
 COM_REDIRECT_FIELD_OLD_URL_DESC="Enter the URL that has to be redirected."
 COM_REDIRECT_FIELD_OLD_URL_LABEL="Expired URL"
-COM_REDIRECT_FIELD_REDIRECT_STATUS_CODE_DESC="Choose the HTTP 1.1 status code to associate with the redirect."
+COM_REDIRECT_FIELD_REDIRECT_STATUS_CODE_DESC="Choose the HTTP 1.1 status code to associate with the redirect. Only status codes from 300-399 correspond to a redirect. "
```
Comment (brianteeman) [inline administrator/language/en-GB/com_redirect.ini]:
```suggestion
COM_REDIRECT_FIELD_REDIRECT_STATUS_CODE_DESC="Choose the HTTP 1.1 status code to associate with the redirect. Only status codes from 300-399 correspond to a redirect."
```

Remove extra space

**Comment by brianteeman** [comment]
Are you saying that if a status code other than 300-399 is selected then there will not be a redirect?

**Comment by PhilETaylor** [comment]
> Are you saying that if a status code other than 300-399 is selected then there will not be a redirect?

**Thats EXACTLY the point.** The Redirects system in Joomla DOESNT REDIRECT if you have advanced mode on and select a code outside 300-399. Then it just returns a RESPONSE with the selected heading. Which is not the purpose of a "redirect" system :-( 

https://github.com/joomla/joomla-cms/issues/32793

File: administrator/language/en-GB/com_redirect.ini (hunk 1 above)
Comment (mrsgv) [reply administrator/language/en-GB/com_redirect.ini]: Removed! Thanks.

**Comment by saumyasarkar11** [comment]
Please add [4.0] to the PR title

**Comment by mrsgv** [comment]
Added! Thanks

**Comment by Quy** [comment]
Let's add the following to `new_url` in `link.xml`:

`showon="header:300,301,302,303,304,305,306,307,308"`

**Comment by PhilETaylor** [comment]
That seems like a better plan, but is not b/c if people are using this bug :-)

**Comment by ghost** [comment]
Shouldn't Issue #32793 be closed or is this pr not good?

**Comment by michwebdev** [comment]
I have tested this item :white_check_mark: successfully on d77e0ce11e185994a6f1266cb175669531b855ee<br /><br />This worked.  I needed to "Activate Advanced Mode" in the Redirects options in order to see the Redirect Status Code option.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32840">issues.joomla.org/tracker/joomla-cms/32840</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d77e0ce11e185994a6f1266cb175669531b855ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32840">issues.joomla.org/tracker/joomla-cms/32840</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32840">issues.joomla.org/tracker/joomla-cms/32840</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Pull request merged**